users/tests/views_test.py

```diff
@@ -86,7 +81,7 @@ def test_post_login_page(self):
         }
 
         response = self.client.post(reverse('login'), creds)
-        self.assertEqual(response.status_code, 302)
+        self.assertEqual(response.status_code, 200)
```

Comment:
wrong test. test if the user has successfully login not if the response is successful